tools/but.py

```diff
@@ -0,0 +1,25 @@
+def list_datasets(butler,collections,dataID,datasetType,where='',max_print=20,
```

Comment:
Please rename this file to something more descriptive, perhaps `butler_tools.py`